RubeesEat/wwwroot/css/site.css

```diff
@@ -113,6 +110,193 @@ nav > ul > li:hover {
     transform: rotate(-45deg) translate(7px, -6px);
 }
 
+p, h1, h2, h3, h4, h5, h6 {
```

Comment:
As you took a lot of stuff from skeleton css. Please add a notice up top that satisfies skeleton CSS's license.

Reply:
According to this, we need to add the MIT License, so I will do that 
https://github.com/dhg/Skeleton?tab=readme-ov-file#license